thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/AnomalyFunctionManager.java

```diff
@@ -25,6 +25,8 @@
 
   List<AnomalyFunctionDTO> findAllByCollection(String collection);
 
+  List<AnomalyFunctionDTO> findAllByMetricAndCollection(String metric, String collection);
```

Comment:
meta: please consider using *findByPredicate()* in the code directly. The DAO interfaces are already overloaded with calls that are used only once.

Reply:
Good point. Switched to findByPredicate